assets/src/edit-story/components/canvas/editElement.js

```diff
@@ -51,8 +57,17 @@ function EditElement({ element }) {
 
   const box = getBox(element);
 
+  useEffect(() => {
+    ref.current.focus();
+  }, []);
+
   return (
-    <Wrapper {...box} onMouseDown={(evt) => evt.stopPropagation()}>
+    <Wrapper
+      ref={ref}
+      tabIndex="0"
```

Comment:
Now that it's tabbable, it really should have some `aria-label`, though I don't really know what it should be.

Maybe we can reuse the `LayerIcon` + `LayerContent` components and `aria-labelledby`?

Reply:
That's a great idea! Done.